modules/service/src/main/scala/ch/epfl/bluebrain/nexus/kg/service/Main.scala

```diff
@@ -48,11 +46,14 @@ object Main {
 
       BootstrapIndexing.startIndexing(settings, bootstrap.sparqlClient, bootstrap.apiUri)
     }
+
+    bootstrap.cluster.joinSeedNodes(bootstrap.seeds.toList)
```

Comment:
If you are controlling the cluster functionality from the Main class, there's no reason to keep the cluster in `BootstrapService`.